ppr-ui/src/views/newMhrRegistration/HomeLocation.vue

```diff
@@ -135,6 +140,24 @@ export default defineComponent({
       }),
       validateLandDetails: computed((): boolean => {
         return !!getSectionValidation(MhrSectVal.LOCATION_VALID, MhrCompVal.LAND_DETAILS_VALID)
+      }),
+      homeLocationCorrection: computed((): UpdatedBadgeIF => {
+        return {
+          baseline: { ...getMhrBaseline.value?.location, address: null },
+          currentState: { ...getMhrRegistrationLocation.value, address: null }
+        }
+      }),
+      civicAddressCorrection: computed((): UpdatedBadgeIF => {
+        return {
+          baseline: getMhrBaseline.value?.location.address,
+          currentState: getMhrRegistrationLocation.value.address
```

Comment:
You might want to use computed for the `currentState`. In my case, when working with Views/Components that do not have badges, I got errors for the `currentState` props being undefined.

Reply:
Oh right sorry, I missed the object itself is computed haha, nevermind!

Reply:
All good!